railties/lib/rails/application/finisher.rb

```diff
@@ -179,6 +174,7 @@ def self.complete(_state)
       # added in the hook are taken into account.
       initializer :set_clear_dependencies_hook, group: :all do |app|
         callback = lambda do
+          # Order matters.
```

Comment:
Could you explain that comment? I'm not sure I get it.

Reply:
Nevermind, I'm blind, I thought the `ActiveSupport::Dependencies.clear` was a second call to `ActiveSupport::DescendantsTracker.clear`.

It all makes sense now that I made a fool of myself :)

Reply:
Heyyy!!! I was on a trip! It happens in the best families! 😀